server/public/styles.css

```diff
@@ -0,0 +1,173 @@
+.hidden {
+  display: none !important;
+}
```

Comment:
Bootstrap has a `d-none` class you can use.
https://getbootstrap.com/docs/4.5/utilities/display/